applications/nrf_desktop/doc/hid_state.rst

```diff
@@ -3,5 +3,174 @@
 HID state module
 ################
 
-HID state module is responsible for aggregating data from user input sources
-and, depending on the state of the connection, transmitting it to the host.
+The |hid_state| is required for generating reports from input data.
+It is responsible for the following operations:
+
+    * Aggregating data from user input sources.
+    * Tracking state of the HID report subscriptions.
+    * Forming the HID reports.
+    * Transmitting the HID reports to the right subscriber.
+
+Module Events
+*************
+
+<TO DO>
+
+Configuration
+*************
+
+The |hid_state| module is enabled by selecting ``CONFIG_DESKTOP_HID_STATE_ENABLE``.
```

Comment:
I don't think it is necessary. The description clearly states what the module does. There is an example usage with mouse & keyboard. I don't think it will serve much purpose.

Reply:
Ok